src/client/SharedQueue.h

```diff
@@ -10,7 +10,7 @@
 #include <thrift/transport/TSocket.h>
 #include <thrift/transport/TTransportUtils.h>
 
-#include "../constants/constants.h"
+#include "constants.h"
 #include "../gen-cpp/RPC.h"
```

Comment:
Anyway we could have the include path be `#include gen-cpp/RPC.h` or `#include RPC.h`

Reply:
That'll be part of a separate PR